dovpanda/config.py

```diff
@@ -5,4 +5,5 @@
 SERIES_CREATION = READ_METHODS + ['Series.__init__']
 
 #
-CATEGORY_SHARE_THRESHOLD = 4
\ No newline at end of file
+CATEGORY_SHARE_THRESHOLD = 4
+MAX_CSV_SIZE = 10000000  # Size in bytes, 10 MB
```

Comment:
10MB is small. I would go with something of at least 100MB. Something that would take around 10 seconds to load